rc.c

```diff
@@ -449,7 +450,7 @@ struct param_ptr params1[] = {
      CMT_EXT_IMAGE_VIEWER, NULL},
     {"image_scale", P_SCALE, PI_TEXT, (void *)&image_scale, CMT_IMAGE_SCALE,
      NULL},
-    {"inline_img_protocol", P_INT, PI_SEL_C, (void *)&enable_inline_image,
+    {"inline_img_protocol", P_CHARINT, PI_SEL_C, (void *)&enable_inline_image_config,
```

Comment:
Why is P_CHARINT needed here?

Reply:
Because P_INT does not accept negative values.

Reply:
Oh, it's ignored when setting the value. I only looked at reading it... Bad naming though... 

But then OK.